### ISSUE TITLE
[Bug] Drawing an area to create an annotation has no visual feedback

### ISSUE THREAD

**Issue by satkowski** [tefkah/zotero-night#52]
Drawing an are in the pdf view is slightly buggy. The drawing functionality still works as expected, however, while I draw an rectangle not visual feedback is given. This means that I don't see the border of the currently drawn rectangle. Only after I release the left mouse button the rectangle is correctly drawn.

I'm using WIndows 10 with the newest version of Zotero (6.0.10) and of this plugin (0.4.15)

**Comment by nhan000** [comment]
Duplicate of https://github.com/tefkah/zotero-night/issues/52, which has been fixed.

**Comment by tefkah** [comment]
thanks!

**Comment by satkowski** [comment]
Thanks for your hard work and this fix!
